pages/project/update-licence-holder/schema/experience-fields.js

```diff
@@ -1,7 +1,23 @@
 const { flattenDeep, castArray } = require('lodash');
 const schema = require('@asl/projects/client/schema/v1/experience').default;
 
-module.exports = version => {
+module.exports = (version, schemaVersion) => {
+  if (schemaVersion === 0) {
+    return {
+      fields: [
+        {
+          name: 'experience-knowledge',
+          label: 'What relevant scientific knowledge or education do you have?',
+          alt: {
+            label: 'What relevant scientific knowledge or education does this person have?'
+          },
+          type: 'texteditor'
+        }
```

Comment:
We _could_ get this from `@asl/projects` but for one field it feels like a giant faff.